collector/lib/CollectorConfig.cpp

```diff
@@ -160,6 +163,48 @@ CollectorConfig::CollectorConfig(CollectorArgs* args) {
   HandleAfterglowEnvVars();
 
   host_config_ = ProcessHostHeuristics(*this);
+
+  const char* envvar;
+
+  connection_stats_quantiles_ = {0.50, 0.90, 0.95};
+
+  if ((envvar = std::getenv("ROX_COLLECTOR_CONNECTION_STATS_QUANTILES")) != NULL) {
+    connection_stats_quantiles_.clear();
+    std::stringstream quantiles(envvar);
+    while (quantiles.good()) {
+      std::string quantile;
+      std::getline(quantiles, quantile, ',');
+      try {
+        double v = std::stod(quantile);
+        connection_stats_quantiles_.push_back(v);
+        CLOG(INFO) << "Connection statistics quantile: " << v;
+      } catch (...) {
+        CLOG(ERROR) << "Invalid quantile value: '" << quantile << "'";
+      }
+    }
+  }
```

Comment:
Probably a good idea to move this to a dedicated `GetConnectionStatsQuantiles()` function in `Utility.cpp` in order to reduce clutter.

Reply:
What do you mean? You can move all of this to a function, including the error handling and return the resulting vector wrapped in a `std::optional` in case the variable is not set.

Reply:
Anyways, you already moved handling of env variables to a separate method, I still think this it quite cluttered, but meh, good enough.